<launch>
    <include file="$(find rocr6_bringup)/launch/usr_bringup.launch" />
    <node pkg="vision_rocr6" type="camera_calibrate_hand_eye" name="camera_calibrate_hand_eye" output="screen">
        <param name="isEyeInHand" type="bool" value="false"/>
        <param name="camera_intrinsic_calibration_url" value="$(find vision_rocr6)/config/intrinsic_calibration.yaml"/>
        <param name="eyeInHand_calibration_url" value="$(find vision_rocr6)/config/eyeInHand_calibration.yaml"/>
        <param name="eyeToHand_calibration_url" value="$(find vision_rocr6)/config/eyeToHand_calibration.yaml"/>
    </node>

</launch>